### PR TITLE
Improve review_checklist.rst

### PR DESCRIPTION
**Pull request by Andersson007**
Improve review_checklist.rst

### CODE REVIEW

**Comment by Andersson007** [comment]
CC @felixfontein @aminvakil

File: review_checklist.rst [hunk 1]
```diff
@@ -29,6 +16,32 @@ Social aspects:
...
   - After merging, thank the author and reviewers for their time and effort.
 
+If you are a committer, also refer to the `Ansible committer guidelines <https://docs.ansible.com/ansible/devel/community/committer_guidelines.html>`_.
+
+Reviewing bug reports
+=====================
+
+When users report bugs, the first thing that is needed is to verify the behaviour.
+
+* [ ] Verify if the user made an obvious mistake in the example code. We often see user errors reported as bugs.
```
Comment (aminvakil) [inline review_checklist.rst:26]:
```suggestion
* [ ] Verify user has made an obvious mistake in the example code. We often see user errors reported as bugs.
```

File: review_checklist.rst [hunk 2]
```diff
@@ -29,6 +16,32 @@ Social aspects:
...
+* [ ] The user does not expect an unexpected behavior. The related documentation is clear.
+* [ ] There is a minimal reproducer. If not, ask the reporter to reduce the complexity to help pinpoint the issue.
+* [ ] The issue is not a consequence of wrong-configured environment.
+* [ ] If it looks a real bug, does the behaviour still exist in the most recent release or the development branch?
```
Comment (aminvakil) [inline review_checklist.rst:30]:
```suggestion
* [ ] If it looks like a real bug, does the behaviour still exist in the most recent release or the development branch?
```

**Review by aminvakil — COMMENTED** [review]
Thanks for improving these @Andersson007 !

File: review_checklist.rst (hunk 1 above)
Comment (Andersson007) [reply review_checklist.rst:26]: @aminvakil I think the sentence is correct (i've just asked a native speaker). I committed the second comment, thank you!

**Comment by Andersson007** [comment]
@aminvakil are the changes OK for you now?:)

**Review by aminvakil — APPROVED** [review]
LGTM

**Comment by Andersson007** [comment]
@aminvakil thanks for reviewing!

**Pull request merged**